opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -1549,7 +1549,7 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
             std::vector<double> volume(this->m_nactive);
 
             #pragma omp parallel for schedule(static)
-            for (std::size_t active_index = 0; active_index < this->m_active_to_global.size(); active_index++) {
+            for (std::int64_t active_index = 0; active_index < static_cast<std::int64_t>(this->m_active_to_global.size()); active_index++) {
```

Comment:
This made me a little curious, what is the reason for this change? Is it due to OpenMP issues?

Reply:
I tried to explain in the commit message.

Reply:
Did not notice the commit message, sorry for the noise!